pkgs/development/compilers/swift/default.nix

```diff
@@ -33,66 +33,73 @@
 }:
 
 let
-  v_base = "5.0.2";
-  version = "${v_base}-RELEASE";
-  version_friendly = v_base;
+  version = "5.1.1";
 
-  tag = "refs/tags/swift-${version}";
   fetch = { repo, sha256, fetchSubmodules ? false }:
     fetchFromGitHub {
       owner = "apple";
       inherit repo sha256 fetchSubmodules;
-      rev = tag;
+      rev = "refs/tags/swift-${version}-RELEASE";
```

Comment:
```suggestion
      rev = "swift-${version}-RELEASE";
```

Reply:
Thanks for catching this. I updated my branch with this change.